mteb/abstasks/AbsTask.py

```diff
@@ -200,7 +200,11 @@ def calculate_metadata_metrics(
 
         descriptive_stats = {}
         hf_subset_stat = "hf_subset_descriptive_stats"
-        pbar_split = tqdm.tqdm(self.metadata.eval_splits, desc="Processing Splits...")
+        eval_splits = self.metadata.eval_splits
+        if self.metadata.type in ["Classification", "MultilabelClassification"]:
+            eval_splits += ["train"]
```

Comment:
Not sure what is going on here?

Reply:
`Classification` and `MultilabelClassification` have an additional train split, so I think it's important to calculate statistics for it as well

Reply:
Ah yes they are sampled - perfect will merge this in then